### DISCUSSION TITLE
Sequoia

### DISCUSSION THREAD

**Discussion by huwpne** [ViennaRSS/vienna-rss#1805] [Q&A]
My version 3.9.2 opens in Sequoia but is not refreshing feeds or displaying web pages - any suggestions

**Comment by barijaona** [comment]
Can you try "Refresh All Subscriptions" and report what details are displayed in Activity window ?

**Comment by huwpne** [comment]
Hi

I have tried the App on a different machine & that is working as expected 

The only difference between the two machines is that one has a VPN running whereas the other is just using Apple’s built in Firewall

IThe VPN Has been disable on there other machine & it’s made no difference - the Activity Log is showing nothing at all
